services/launchpad/launchpad.tester.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('../service-tester');
+
+const t = new ServiceTester({ id: 'launchpad', title: 'Launchpad.net' })
+module.exports = t;
+
+t.create('gimp ppa version')
+  .get('/ppa/version/otto-kesselgulasch/gimp/gimp.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'launchpad',
+    value: Joi.string().regex(/^\d\.\d/)
```

Comment:
Could you update this to use `\d+` incase the version is something like `12.3.4`